clang/test/dpct/helper_files_ref/include/util.hpp

```diff
@@ -466,6 +466,59 @@ nd_range_barrier(cl::sycl::nd_item<1> item,
 
   item.barrier();
 }
+
+/// The logical-group is a logical collection of some work-items within a
+/// work-group. Each logical-group size is a power of 2 in the range
+/// [1, current_sub_group_size]. If the logical-group size is illegal,
```

Comment:
it's caller's respond to make sure logical-group size is in the range,  and is a number =2**N.  (device code has not way to through exception here).

Reply:
OK, I will remove the "if" block.

Reply:
done